pkg/kind/cluster.go

```diff
@@ -70,7 +72,19 @@ func (c *Cluster) getConfig() ([]byte, error) {
 	var err error
 
 	if c.kindConfigPath != "" {
-		rawConfigTempl, err = os.ReadFile(c.kindConfigPath)
+		if strings.HasPrefix(c.kindConfigPath, "https://") || strings.HasPrefix(c.kindConfigPath, "http://") {
+			resp, err := http.Get(c.kindConfigPath)
```

Comment:
Let's add a timeout. There is a utility function to get the http client we use for reconciliation. Maybe we can use that. Note that the timeout value is configured low because we expect endpoints to be locally available.